package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@reduxjs/toolkit": "^1.8.3",
```

Comment:
redux tookkit is present in package.json 🙌

Reply:
Yeah so, what does this mean? is it good or bad?